source/globals.d.ts

```diff
@@ -1,2 +1,85 @@
 // TODO: Type anything that is of type AnyObject
-type AnyObject = Record<string, any>;
+type AnyObject = Record<string, any> | undefined;
+
+declare module "github-reserved-names"; // TODO: PR types to that repo or DT
+
+// TODO: PR these types to the delegate repository.
+
+declare module "delegate" {
+	// Custom event types used in RGH
+	type EventType = keyof GlobalEventHandlersEventMap | 'details:toggled';
+}
+
+// TODO: PR this into the delegate repo.
+declare module "delegate" {
```

Comment:
@bfred-it, I check the API and documentation in `delegate` and check the types I created here in RGH and they all seem to resolve the types correctly. Just a give it a glance and then I'll PR it to the `delegate` repo which I believe you're a maintainer of?

Reply:
Sadly `delegate` hasn't accepted my October 2018 PRs yet

Reply:
@bfred-it Fork it 💪

Reply:
Let's wait for this first https://twitter.com/bfred_it/status/1097442284839157760

Reply:
Done. @nickytonline you can contribute the types in https://github.com/bfred-it/delegate-it/issues/1

Reply:
Will do. I'll PR something up for you as we want the types to be a bit more strict.